src/foam/u2/crunch/CrunchController.js

```diff
@@ -84,6 +85,13 @@ foam.CLASS({
         return new Map();
       }
     },
+    {
```

Comment:
Could you refine this in the DebugWizardView.

Reply:
This would leave the implementation of `createWizardSequence` open to crashing (in case appConfig doesn't match). I have an idea to create a subclass of Method called SequenceMethod - if I do that I could do something like this in the refinement:

```
methods: [
  foam.u2.crunch.CrunchController.CREATE_WIZARD_SEQUENCE.clone()
    .add(foam.u2.crunch.wizardflow.DebugAgent)
]
```